src/atoms/Calendar/week/WeekCalendar.tsx

```diff
@@ -130,50 +117,80 @@ export const WeekCalendar: FC<WeekCalendarProps> = ({
     weekStartsOn
   ]);
 
+  const chevronClasses = clsx(
+    'flex flex-none align-center justify-center',
+    'text-primary size-8 font-bold'
+  );
+
+  const cannotActivate = (day: (typeof days)[0]) => {
```

Comment:
@bejoinka @sharlotta93 This could use a useMemo as this variable now be computed on every re-render.